### ISSUE TITLE
Vanilla CO Function Follow-Up

### ISSUE THREAD

**Issue by TheMasterCreed** [Robosturm/Commander_Wars#503]
Just following up from our discussion from Discord as to requesting you to implement a mod that makes all CO's function as they would from their own respective games.

![image](https://user-images.githubusercontent.com/85777134/126834874-787b364c-ae3e-4971-84e3-2464303886bd.png)

So now you have me really curious and I'd love to hear what you have to say and if you would implement such a request.

**Comment by Robosturm** [comment]
Personally i think a mod like this is more about testing out how AW1-3 CO's perform against AW-DC CO's but it's nothing that a lot of people would play (I might be wrong here).
Also in most case global bonuses will win against a local boost.
I'll consider to add it when i run out of ideas to add to the game.
But currently there's a large list of features i prefer to work on instead.
So don't expect that something pop's up here in the near future. ;)

**Comment by TheMasterCreed** [comment]
I'll just use my mod for the time being then!
Primarily I wanted to test out Caulder Vs. Sturm when I had originally
wanted such a feature. He's the only CO in the entire series that still
might be the most broken CO even with only a local boost. I got a YouTuber
called Mangs interested in attempting some matches between the two, and if
it's even possible for Sturm to pull out on top.

It's okay if you won't officially support it, my edits do what's needed,
all I did was add the Darc Conflict CO mod script to all of the DoR CO's,
and the Duel Strike CO mod scripts were added to the other CO's, along with
disabling the CO unit action for the CO's not from DoR.
